src/Raven.Server/RavenServer.cs

```diff
@@ -1006,18 +1006,17 @@ public bool RefreshClusterCertificate(object state, string raftRequestId)
             var forceRenew = state as bool? ?? false;
 
             var currentRefreshTask = _currentRefreshTask;
-            if (currentRefreshTask.IsCompleted == false)
+            if (currentRefreshTask.Unwrap().IsCompleted == false)
```

Comment:
What will this do if the task hasn't been resolved yet?
Maybe:

```suggestion
            if (currentRefreshTask.IsCompleted == false || currentRefreshTask.Unwrap().IsCompleted == false)
```